modules/telco-ref-design-overview.adoc

```diff
@@ -1,9 +1,14 @@
 // Module included in the following assemblies:
 //
-// *
+// * scalability_and_performance/telco_ran_du_ref_design_specs/telco-ran-du-rds.adoc
+// * scalability_and_performance/telco_ref_design_specs/telco-ref-design-specs-overview.adoc
+
 :_mod-docs-content-type: CONCEPT
 [id="telco-ref-design-overview_{context}"]
-= Reference design specifications for telco 5G deployments
+= Reference design specifications for telco RAN DU 5G deployments
+
+The reference configurations in this document are deployed using a centrally managed hub cluster infrastructure as shown in the following image.
+The reference configuration includes the configuration of the far edge clusters and hub cluster components.
```

Comment:
This paragraph leads into the diagram below (line 23) but the intervening 2 paragraphs are more general. Maybe consider moving this to just before the image?